hoot-core/src/main/cpp/hoot/core/algorithms/zindex/ZCurveRanger.cpp

```diff
@@ -361,7 +361,7 @@ vector<Range> ZCurveRanger::_decomposeRangeIterative(LongBox box, int count)
       }
       else
       {
-        throw new HootException("Invalid boxes.size = " + boxes.size());
+        throw HootException("Invalid boxes.size = " + boxes.size());
```

Comment:
`HootException` created on the heap and then thrown and never disposed of when caught.